contents/docs/data/persons.mdx

```diff
@@ -60,9 +63,21 @@ client.capture({
 })
 ```
 
+### Person profiles and distinct IDs
+
+Below is good information to know when working with distinct IDs and person profiles:
+
+- Every event has a `distinct_id` representing a user attached to it. However, not every `distinct_id` has an associated person profile.
+
+- If a distinct ID already has a person profile created, any events captured with this distinct ID are considered events with a person profile. 
+    - To opt out of this behavior, [delete the person](#deleting-person-data) in the person details UI in PostHog or use the `DELETE` operation of the [person API endpoint](/docs/api/persons#delete-api-projects-project_id-persons-id).
```

Comment:
We should address the not reusing `distinct_id` point: https://posthog.com/questions/why-do-i-see-person-not-found-for-a-user-that-gets-recreated-after-deletion (just realized this might be inconsistent)

Asked in Slack: https://posthog.slack.com/archives/C06R39WNY5B/p1717774463796589